test/e2e/framework.go

```diff
@@ -740,11 +749,7 @@ func (data *TestData) createPodOnNode(name string, nodeName string, image string
 	}
 	if nodeName == controlPlaneNodeName() {
 		// tolerate NoSchedule taint if we want Pod to run on control-plane Node
-		noScheduleToleration := corev1.Toleration{
-			Key:      labelNodeRoleControlPlane(),
-			Operator: corev1.TolerationOpExists,
-			Effect:   corev1.TaintEffectNoSchedule,
-		}
+		noScheduleToleration := controlPlaneNoScheduleToleration()
```

Comment:
Should this line of code be changed as well?
https://github.com/vmware-tanzu/antrea/blob/main/test/e2e/performance_test.go#L52